cmd/kubeadm/app/master/clusterroles.go

```diff
@@ -0,0 +1,87 @@
+package master
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+	apiv1 "k8s.io/kubernetes/pkg/api/v1"
+	rbac "k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+)
+
+// CreateBootstrapRBACClusterRole creates the necessary ClusterRole for bootstrapping
+func CreateBootstrapRBACClusterRole(clientset *clientset.Clientset) error {
+	clusterRole := rbac.ClusterRole{
+		ObjectMeta: apiv1.ObjectMeta{Name: "kubeadm:kubelet-bootstrap"},
```

Comment:
Still wanna poke this (can be a follow-up as well), @liggitt @deads2k Why can't this rule be a `system:` rule for the TLS bootstrap? Nothing here is kubeadm-specific, it's TLS Bootstrap specific which seems like a core thing.

Or is this just the same as the `system:discovery` RoleBinding described here: https://github.com/kubernetes/kubernetes.github.io/pull/2169?